common/lib/capa/capa/inputtypes.py

```diff
@@ -592,8 +592,11 @@ def get_attributes(cls):
                                              #   set state
             Attribute('width', "400"),       # iframe width
             Attribute('height', "300"),      # iframe height
-            Attribute('sop', None)           # SOP will be relaxed only if this
-                                             # attribute is set to false.
+            # Title for the iframe, should be supplied by the author of the problem. Not translated
+            # because default display name is also not translated, and we can't import Django here anyway.
```

Comment:
My understanding (which certainly could be wrong!) is that translation can't happen here because it is a classmethod. See also this block of pre-existing code in the file:
```
@classmethod
    def get_attributes(cls):
        # Make '_' a no-op so we can scrape strings. Using lambda instead of
        #  `django.utils.translation.ugettext_noop` because Django cannot be imported in this file
```

Translation is possible for instance methods where the runtime can be used, for example:

```
def setup(self):
        _ = self.capa_system.i18n.ugettext
        submitted_msg = _("Your files have been submitted. As soon as your submission is"
                          " graded, this message will be replaced with the grader's feedback.")
        self.submitted_msg = submitted_msg
```
I can update the comment to say that it is more the fact that it is a classmethod, and therefore capa_system.i18n is not available (and since Django can't be imported, translation can't happen using that mechanism either).